slycot/tests/test.py

```diff
@@ -64,6 +64,7 @@ def test_td04ad_static(self):
             nr,a,b,c,d = transform.td04ad(rc,nin,nout,index,den,num)
 
 
+
```

Comment:
a nitpick: please remove this new newline, then this file won't be in the changeset at all.